src/main.js

```diff
@@ -246,9 +255,16 @@ function run(commands) {
     });
 }
 
+function combineSourceStreams(context, sourceStreams, shouldGroupOutput) {
+    if (shouldGroupOutput) {
+        return Rx.Observable.concat.apply(this, sourceStreams);
+    }
+    return Rx.Observable.merge.apply(this, sourceStreams);
+}
+
 function handleOutput(streams, childrenInfo, source) {
     var sourceStreams = _.map(streams, source);
-    var combinedSourceStream = Rx.Observable.merge.apply(this, sourceStreams);
+    var combinedSourceStream = combineSourceStreams(this, sourceStreams, config.group);
```

Comment:
Though the config is available as a global, I chose to pass it as a parameter to this new function. I prefer this style, but I'm open to changing it